### ISSUE TITLE
Build failing with Python 3.13

### ISSUE THREAD

**Issue by kmaehashi** [scoder/fastrlock#20]
One of the CuPy user reported that there is an error building fastrlock on Python 3.13. After the quick investigation, this looks happening because `rlock.c` bundled with the sdist is generated using older Cython which does not support Python 3.13.

```
% pip install -v fastrlock

... snip ...

  fastrlock/rlock.c: In function ‘__Pyx_PyInt_As_long’:
  fastrlock/rlock.c:9635:23: error: too few arguments to function ‘_PyLong_AsByteArray’
   9635 |                 ret = _PyLong_AsByteArray((PyLongObject *)v,
        |                       ^~~~~~~~~~~~~~~~~~~
  In file included from /home/maehashi/.pyenv/versions/3.13.1/include/python3.13/longobject.h:107,
                   from /home/maehashi/.pyenv/versions/3.13.1/include/python3.13/Python.h:81,
                   from fastrlock/rlock.c:35:
  /home/maehashi/.pyenv/versions/3.13.1/include/python3.13/cpython/longobject.h:111:17: note: declared here
    111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
        |                 ^~~~~~~~~~~~~~~~~~~
  fastrlock/rlock.c: In function ‘__Pyx_PyInt_As_int’:
  fastrlock/rlock.c:9908:23: error: too few arguments to function ‘_PyLong_AsByteArray’
   9908 |                 ret = _PyLong_AsByteArray((PyLongObject *)v,
        |                       ^~~~~~~~~~~~~~~~~~~
  In file included from /home/maehashi/.pyenv/versions/3.13.1/include/python3.13/longobject.h:107,
                   from /home/maehashi/.pyenv/versions/3.13.1/include/python3.13/Python.h:81,
                   from fastrlock/rlock.c:35:
  /home/maehashi/.pyenv/versions/3.13.1/include/python3.13/cpython/longobject.h:111:17: note: declared here
    111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
        |                 ^~~~~~~~~~~~~~~~~~~
  error: command '/home/maehashi/local/bin/ccache-gcc' failed with exit code 1

... snip ...
```

It would be great if we can see a new release including the source generated with the latest Cython (or binary packages for Python 3.13).

**Comment by leofang** [comment]
+1 it's be nice to help unblock CuPy (which in turn would unblock us 😄) 

cc @szalpal @aterrel for vis

**Comment by scoder** [comment]
I released fastrlock 0.8.3 with wheels up to Py3.13.

**Comment by kmaehashi** [comment]
Thank you!
